src/main/java/com/cachinginapp/enterprise/dao/CacheDAOStub.java

```diff
@@ -21,4 +21,13 @@ public Cache save(Cache cache) throws Exception {
     public List<Cache> fetchAll() {
         return allCaches;
     }
+    @Override
+    public Cache fetchCacheById(int id) {
```

Comment:
use a fetchCacheById for better results

Reply:
This was covered in a previously submitted code review, from another student.  When that occurs, first in wins.